index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Home | CSCS</title>
+    <link rel="stylesheet" href="https://rawcdn.githack.com/anonymoustaco/collegestationcodesociety.github.io/b57ed23d7334cfb26ae5095e43e279a0c4563211/main.css">
+    <link rel="icon" type="image/gif" href="https://rawcdn.githack.com/anonymoustaco/collegestationcodesociety.github.io/0cfb50ba59b0cdd51b391f1c884a2f1681e31f74/favicon.gif" />
+</head>
+<body>
+    <div class="nav-container">
+       <img id="logo" src="https://rawcdn.githack.com/anonymoustaco/collegestationcodesociety.github.io/0cfb50ba59b0cdd51b391f1c884a2f1681e31f74/Black_logo.png" />
```

Comment:
```suggestion
       <img id="logo" src="Black_logo.png" />
```

Reply:
One thing these changes will require is that you include the related files in your pull request. Go ahead and do that.